src/backend/legendary/games.ts

```diff
@@ -614,10 +614,12 @@ class LegendaryGame extends Game {
           this.logFileLocation,
           `Launch aborted: ${wineLaunchPrepFailReason}`
         )
-        showErrorBoxModalAuto({
-          title: t('box.error.launchAborted', 'Launch aborted'),
-          error: wineLaunchPrepFailReason!
-        })
+        if (wineLaunchPrepFailReason) {
```

Comment:
Added this check instead of `!` since this param can actually be undefined in many cases and it was throwing an error in the frontend and killing the react app

Reply:
Looks like *someone* (me) didn't check this properly. Sorry!

Having *some* kind of message, even if no failure reason was given, might still be a good idea

Reply:
In this case I'm already displaying an error dialog before this, I think it's fine to make this optional or it would try to display 2 errors messages (one for the missing wine files and then one here).

Reply:
Ah right, now I remember how I originally planned this function... Either you show an error message yourself (that can be more specific/include buttons and so on), or you set `failureReason` and get the generic "Launch aborted" message